src/main/java/de/unistuttgart/iste/sqa/pse/sheet11/presence/calculator/Calculator.java

```diff
@@ -28,16 +28,14 @@
 	 */
 	public int add(final int summand1, final int summand2);
 
-	/*
-	 * @requires divisor != Interger.MIN_VALUE
-	 * 
-	 * @requires divisor != 0
-	 * 
-	 * @requires dividend != Integer.MIN_VALUE
-	 * 
-	 * @ensures divisor != 0 ==> \result = correctly rounded quotient
-	 * 
-	 */
+	/*@
+	 @ requires divisor != Integer.MIN_VALUE;
+	 @ requires divisor != 0;
+	 @ requires dividend != Integer.MIN_VALUE;
+	 @
+	 @ ensures divisor != 0 ==> \result == correctly rounded quotient;
```

Comment:
Postconditions like this seem to be unnecessarily complex. The postcondition checks the precondition again, although the precondition must hold by definition.
```suggestion
	 @ ensures \result == correctly rounded quotient;
```